.github/matrix.json

```diff
@@ -1,5 +1,6 @@
 {
   "postgresql": ["12"],
   "ruby": ["2.7"],
-  "node": ["14"]
+  "node": ["16"],
+  "npm": ["6"]
```

Comment:
Can this get updated to 8 so we can see the tests run?

Reply:
with 8 most tests will fail as it cant find most packages correctly, if it even runs npm i succefully

Reply:
adding legacy-peer-deps to npmrc and then maybe we can just use the newer npm without fixing anything